testpackage/tests/Ionosphere_small/Ionosphere_small.cfg

```diff
@@ -148,6 +148,7 @@ constBgBX = 0.0
 constBgBY = 0.0
 constBgBZ = -5.0e-9
 noDipoleInSW = 0 # Use 0 with dipole type 4
+dipoleScalingFactor = 0.1 # smaller dipole for faster testing
```

Comment:
Is that helping with the radius too then? We/I've procrastinated that other PR on this for a while.

Reply:
I didn't change radius, but thought that to get any valid ionosphere output we need a longer run. Now runs to 30 seconds instead of 3 seconds.
```
Ionosphere_small 
------------------------------------------------------------
 ref-time     |   new-time       |  speedup                |
------------------------------------------------------------
125.2        271.9         0.460463
------------------------------------------------------------

Reply:
OK then this guy's downmap radius (6.5 RE) is still within the inner radius of 70000 km. Weakening the dipole should allow to pull in the radius enough, or conversely push out the downmap radius and still map. Can be handled as part of the separate ionosphere radius tp PR.

Reply:
In https://github.com/fmihpc/vlasiator/pull/797 I wrote:
> I've made a test run at `/wrk-vakka/users/markusb/KTH/Ionosphere_intermediate` with 30 seconds of simulation, FS subcycling, the inner boundary at 25e6 metres (just under 4 RE), and a dipole scaling factor of 0.001. This ran in 756 seconds on Carrington so is still relatively expensive for CI. I'm also a bit surprised that FS subcycles were consistently at the maximum allowed value. Up for discussion and adjustment.

An issue with pulling the dipole magnitude down a lot (like to 0.1%) is that then the shock does not really form as the obstacle is too weak. And despite that, pushing the inner boundary inwards still took a lot of time to simulate. 

This is the state at 15s:
![image](https://github.com/fmihpc/vlasiator/assets/11755746/52ceee74-c0fd-4e72-b05e-7bf6b1aebe24)

But yea, getting ionospheric outputs which can actually be evaluated is too much for this PR.